README.md

```diff
@@ -12,8 +12,8 @@ This proyect is in active development. Do expect bugs and general trouble, and p
 
 1. Clone this repository and change directory to it:
    ```
-   $ git clone https://github.com/sfultong/stand-in-language.git
```

Comment:
We should probably change to https://github.com/Stand-In-Language/stand-in-language.git